src/pages/index.tsx

```diff
@@ -35,6 +70,23 @@ const Index: FC = () => {
     }
   }, [isLoading, response]);
 
+  useEffect(() => {
+    const url = SELF_URL;
+    const fetchData = async () => {
+      try {
+        const r_esponse = await fetch(url);
+        const json = await r_esponse.json();
```

Comment:
Use our own [fetch helper function](https://github.com/Real-Dev-Squad/website-status/blob/develop/src/helperFunctions/fetch.ts)

Reply:
Is this the right way?
```useEffect(() => {
    const url = SELF_URL;
    const fetchData = async () => {
      try {
        const r_esponse = await fetch({ url });
        const adminUser :boolean = r_esponse.data.user.roles.admin;
        const superUser :boolean = r_esponse.data.user.roles.super_user;
        if (adminUser || superUser === true) {
          setRoles(true);
        }
      } catch (error_) {
        // eslint-disable-next-line no-console
        console.log('error', error_);
      }
    };

    fetchData();
  }, []);

Reply:
Use [object destructuring](https://javascript.info/destructuring-assignment#object-destructuring)

Reply:
Any reason for `const url = SELF_URL`?

Reply:
I initialized this url globally and thought it would be better to use it this way rather than writing the whole url again.
I could just use it without creating another variable also. I will do that

Reply:
Also what is with this `snake_case` variable `r_esponse`?
We strictly use `camelCase` for our variables.

Reply:
Alright I will sort it out